common.go

```diff
@@ -34,12 +32,6 @@ func getCopyOptions(reportWriter io.Writer, sourceReference types.ImageReference
 
 		}
 	}
-	sourceInsecure, err := isReferenceInsecure(sourceReference, sourceCtx)
-	if err != nil {
-		logrus.Debugf("error determining if registry for %q is insecure: %v", transports.ImageName(sourceReference), err)
-	} else if sourceInsecure {
-		sourceCtx.OCIInsecureSkipTLSVerify = true
```

Comment:
For the docker transport, yes. But c/image will do this automatically now (unless it's specifically set by the user, e.g., via the CLI flag).

Reply:
`DockerInsecureSkipTLSVerify` used to be set here before https://github.com/containers/buildah/pull/1214 , which dropped it to rely on the automatic c/image/docker support.

#1214 did not remove the `OCIInsecureSkipTLSVerify` part because we did not have anything equivalent in c/image/oci, this PR drops it, because on further review it can never execute.